### PR TITLE
added known issue for global hub

### PR DESCRIPTION
**Pull request by jc-berger**
See Jira: https://issues.redhat.com/browse/ACM-10446

### CODE REVIEW

File: release_notes/known_issues_global.adoc [hunk 1]
```diff
@@ -23,6 +23,14 @@ Or consider a troubleshooting topic.
 
 Review the known issues for the {global-hub} Operator. The following list contains known issues for this release, or known issues that continued from the previous release. For your {ocp-short} cluster, see link:https://access.redhat.com/documentation/en-us/openshift_container_platform/4.12/html/release_notes/ocp-4-12-release-notes#ocp-4-12-known-issues[{ocp-short} known issues].
 
+[#kafka-operator-keeps-restarting]
+== Kafka operator keeps restarting 
+
+In the Federal Information Processing Standard (FIPS) environment, the Kafka operator keeps restarting because of its out-of-memory (OOM) state. To fix this issue, complete the following steps:
+
+. Set the resource limit to at least `512M`.
```
Comment (jc-berger) [inline release_notes/known_issues_global.adoc]: Am I understanding correctly that it's a two step process like I've documented here?

Comment (ldpliu) [reply release_notes/known_issues_global.adoc]: Thanks for your quick response, it's not two steps, it's one step, and the link is the detail description about this step. So the line 31-32 should be : 
`Set the kafka operator resource limit to at least to `512M`, details please see: https://access.redhat.com/documentation/en-us/red_hat_amq_streams/2.6/html/deploying_and_managing_amq_streams_on_openshift/deploy-intro_str#assembly-fips-support-str[amq stream doc]`

**Review by ldpliu — APPROVED** [review]

File: release_notes/known_issues_global.adoc [hunk 2]
```diff
@@ -23,6 +23,11 @@ Or consider a troubleshooting topic.
 
 Review the known issues for the {global-hub} Operator. The following list contains known issues for this release, or known issues that continued from the previous release. For your {ocp-short} cluster, see link:https://access.redhat.com/documentation/en-us/openshift_container_platform/4.12/html/release_notes/ocp-4-12-release-notes#ocp-4-12-known-issues[{ocp-short} known issues].
 
+[#kafka-operator-keeps-restarting]
+== Kafka operator keeps restarting 
+
+In the Federal Information Processing Standard (FIPS) environment, the Kafka operator keeps restarting because of its out-of-memory (OOM) state. To fix this issue, set the resource limit to at least `512M`. For detailed steps on how to set this limit, see link:https://access.redhat.com/documentation/en-us/red_hat_amq_streams/2.6/html/deploying_and_managing_amq_streams_on_openshift/deploy-intro_str#assembly-fips-support-str[amq stream doc].
```
Comment (swopebe) [inline release_notes/known_issues_global.adoc]: Remove its, try not to use possessives when you can avoid it. This is in the manuals.

**Review by swopebe — APPROVED** [review]
See comment about possessives, which can be reviewed in manuals.

**Pull request merged**